e2e/e2etests/e2etests.go

```diff
@@ -533,6 +534,12 @@ var AllE2ETests = []runner.E2ETest{
 		[]runner.ArgDefinition{},
 		TestRateLimiter,
 	),
+	runner.NewE2ETest(
+		TestCriticalAdminTransactionsName,
+		"test critical admin transactions",
+		[]runner.ArgDefinition{},
+		TestAdminTransactions,
```

Comment:
Function name should be consistent with test name, use critical or not in the name